js/index.js

```diff
@@ -3,7 +3,7 @@ let today = new Date();
 let thisYear = today.getFullYear();
 let footer = document.querySelector('.footer');
 let copyright = document.createElement('p')
-copyright.innerHTML = "Emmely Suarez " + thisYear;
+copyright.innerHTML =  "\u00a9 " +"Emmely Suarez " + thisYear;
```

Comment:
A small thing, but since it's easy to forget codes, using a named entity is another option:
```suggestion
copyright.innerHTML =  "&copy; Emmely Suarez " + thisYear;
```